DESCRIPTION

```diff
@@ -37,7 +38,24 @@ Imports:
     dplyr
 Suggests: 
     testthat (>= 3.0.0),
-    covr
+    covr,
+    rmarkdown,
+    knitr
 Config/testthat/edition: 3
 Depends: 
     R (>= 2.10)
+Collate: 
+    'annotate.R'
+    'battles.R'
+    'class.R'
+    'coerce_from_messydate.R'
+    'coerce_to_messydate.R'
+    'contract.R'
+    'expand.R'
+    'extract.R'
+    'logical.R'
+    'messydate_make.R'
+    'operate.R'
+    'resequence.R'
+    'resolve.R'
+    'set.R'
```

Comment:
Necessary?

Reply:
This was added automatically for some reason... I do not think it is needed, so I will remove it.